test/cloud_test.go

```diff
@@ -0,0 +1,97 @@
+// The MIT License
+//
+// Copyright (c) 2020 Temporal Technologies Inc.  All rights reserved.
+//
+// Copyright (c) 2020 Uber Technologies, Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+package test_test
+
+import (
+	"context"
+	"flag"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+	"go.temporal.io/api/cloud/cloudservice/v1"
+	"go.temporal.io/sdk/client"
+)
+
+var cloudTestsFlag bool
+
+func init() { flag.BoolVar(&cloudTestsFlag, "cloud-tests", false, "Enable cloud tests") }
+
+func TestCloudSuite(t *testing.T) {
```

Comment:
Nit: Can we rename this to something like `TestCloudOpsSuit` or similar to clarify these tests are just for the clouds Ops API.

Reply:
I can change this. I was actually thinking that this could encompass all cloud-only tests and not just operations ones (granted it's only operations tests today that are cloud only).

Reply:
Changed

Reply:
What kind of tests would be cloud-only that aren't the cloud ops tests?

We have other tests that already run against cloud https://github.com/temporalio/sdk-go/actions/runs/9784335348/job/27014986110, but I guess they are not cloud only.

Reply:
I am not sure, but we can refactor when we get there (maybe we'll never there)